sources/packages/web/src/views/student/application-offering-change-request/RequestedApplicationOfferingDetails.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <body-header-container>
+    <template #header>
+      <body-header :title="changeRequest.studentFullName">
```

Comment:
I was thinking it will be better to take the student name from the store, rather than getting it from API on every load.
thoughts @dheepak-aot @guru-aot @andrepestana-aot

Reply:
I agree. For the sake of getting student name we are joining the student and users in API side which can be avoided.

Reply:
I have made this change on the web side and added the student's `fullName` to the store for the student. But I haven't changed the API since the same api call also fetches the data for the `RequestAChangeFormView.vue` used in the institution.

Reply:
for the institution the controller is different right, so Which means the DTO will be different for both controllers, so can't be remove the student name from the student controller? unless I am missing something?